radicle-seed/Cargo.toml

```diff
@@ -0,0 +1,20 @@
+[package]
+name = "radicle-seed"
+version = "0.1.0"
+authors = ["The Radicle Team <dev@radicle.xyz>"]
+edition = "2018"
+license = "GPL-3.0-or-later"
+
+[dependencies]
+librad = { path = "../librad" }
+async-trait = "0.1"
+thiserror = "1.0"
+tokio = "0.2"
+futures = "0.3"
+argh = "0.1.3"
+log = { version = "0.4", features = ["std"] }
```

Comment:
Most likely you'll want to use [tracing](https://crates.io/crates/tracing) / [tracing-subscriber](https://crates.io/crates/tracing-subscriber) instead, to be able to debug the lower bowels.

Reply:
Done.